RELEASES.md

```diff
@@ -1,3 +1,122 @@
+Version 1.39.0 (2019-11-07)
+===========================
+
+Language
+--------
+- [You can now create `async` functions and blocks with `async fn`, `async move {}`, and
+  `async {}` respectively, and you can now call `.await` on async expressions.][63209]
+- [You can now use `cfg` and lint (e.g. `warn`, `deny`, `allow`) attributes on
```

Comment:
You can also use inert attributes on function parameters that proc macros can use.

Reply:
How would you explain that briefly without confusing it with allowing proc macro attributes?

Reply:
> You can now use certain attributes on function, closure, and function pointer parameters.
> These attributes include `cfg`, `cfg_attr`, `allow`, `warn`, `deny`, `forbid` as well as inert helper
> attributes used by procedural macro attributes applied to items.

Reply:
cc @petrochenkov ^--